docs/09-bootstrapping-kubernetes-workers.md

```diff
@@ -199,6 +201,7 @@ ExecStart=/usr/local/bin/kubelet \\
   --register-node=true \\
   --require-kubeconfig \\
   --runtime-request-timeout=10m \\
+  --client-ca-file=/var/lib/kubernetes/ca.pem \\
```

Comment:
Give the kubelet the ability to validate certificates against the CA to know the ```subject``` to send for SubjectAccessReview.